dpnp/backend/include/dpnp_gen_1arg_2type_tbl.hpp

```diff
@@ -85,95 +85,15 @@
-                     sycl::atan(input_elem),
-                     oneapi::mkl::vm::atan(q, input1_size, input1_data, result))
-MACRO_1ARG_2TYPES_OP(
-    dpnp_atanh_c,
-    sycl::atanh(input_elem),
-    oneapi::mkl::vm::atanh(q, input1_size, input1_data, result))
-MACRO_1ARG_2TYPES_OP(dpnp_cbrt_c,
-                     sycl::cbrt(input_elem),
-                     oneapi::mkl::vm::cbrt(q, input1_size, input1_data, result))
-MACRO_1ARG_2TYPES_OP(dpnp_ceil_c,
-                     sycl::ceil(input_elem),
-                     oneapi::mkl::vm::ceil(q, input1_size, input1_data, result))
 MACRO_1ARG_2TYPES_OP(dpnp_copyto_c, input_elem, q.submit(kernel_func))
-MACRO_1ARG_2TYPES_OP(dpnp_cos_c,
-                     sycl::cos(input_elem),
-                     oneapi::mkl::vm::cos(q, input1_size, input1_data, result))
-MACRO_1ARG_2TYPES_OP(dpnp_cosh_c,
-                     sycl::cosh(input_elem),
-                     oneapi::mkl::vm::cosh(q, input1_size, input1_data, result))
 MACRO_1ARG_2TYPES_OP(dpnp_degrees_c,
                      sycl::degrees(input_elem),
                      q.submit(kernel_func))
-MACRO_1ARG_2TYPES_OP(dpnp_exp2_c,
-                     sycl::exp2(input_elem),
-                     oneapi::mkl::vm::exp2(q, input1_size, input1_data, result))
-MACRO_1ARG_2TYPES_OP(dpnp_exp_c,
-                     sycl::exp(input_elem),
-                     oneapi::mkl::vm::exp(q, input1_size, input1_data, result))
-MACRO_1ARG_2TYPES_OP(
-    dpnp_expm1_c,
-    sycl::expm1(input_elem),
-    oneapi::mkl::vm::expm1(q, input1_size, input1_data, result))
-MACRO_1ARG_2TYPES_OP(dpnp_fabs_c,
-                     sycl::fabs(input_elem),
-                     oneapi::mkl::vm::abs(q, input1_size, input1_data, result))
-MACRO_1ARG_2TYPES_OP(
-    dpnp_floor_c,
-    sycl::floor(input_elem),
-    oneapi::mkl::vm::floor(q, input1_size, input1_data, result))
-MACRO_1ARG_2TYPES_OP(
-    dpnp_log10_c,
-    sycl::log10(input_elem),
-    oneapi::mkl::vm::log10(q, input1_size, input1_data, result))
-MACRO_1ARG_2TYPES_OP(
-    dpnp_log1p_c,
-    sycl::log1p(input_elem),
-    oneapi::mkl::vm::log1p(q, input1_size, input1_data, result))
-MACRO_1ARG_2TYPES_OP(dpnp_log2_c,
-                     sycl::log2(input_elem),
-                     oneapi::mkl::vm::log2(q, input1_size, input1_data, result))
-MACRO_1ARG_2TYPES_OP(dpnp_log_c,
-                     sycl::log(input_elem),
-                     oneapi::mkl::vm::ln(q, input1_size, input1_data, result))
 MACRO_1ARG_2TYPES_OP(dpnp_radians_c,
                      sycl::radians(input_elem),
                      q.submit(kernel_func))
-MACRO_1ARG_2TYPES_OP(dpnp_sin_c,
-                     sycl::sin(input_elem),
-                     oneapi::mkl::vm::sin(q, input1_size, input1_data, result))
-MACRO_1ARG_2TYPES_OP(dpnp_sinh_c,
-                     sycl::sinh(input_elem),
-                     oneapi::mkl::vm::sinh(q, input1_size, input1_data, result))
 MACRO_1ARG_2TYPES_OP(dpnp_sqrt_c,
```

Comment:
Please leave a comment why we have to keep `dpnp_sqrt_c` as for now (point that it is used by another function)